### ISSUE TITLE
RSS feed is missing (correct) link and link[rel=self] tags

### ISSUE THREAD

**Issue by fwenzel** [bigboringsystem/bigboringsystem#200]
Looks like the feed needs to have a teeny tiny more metadata attached to it:

https://en.wikipedia.org/wiki/Atom_%28standard%29#Example_of_an_Atom_1.0_feed

Right now, it contains:
``<link>http://github.com/dylang/node-rss</link>``
which is obviously not what we want.

**Comment by coleGillespie** [comment]
@fwenzel eslint breaks when you add this meta data currently. there is a fix going into the rss module hopefully soon to fix this. [see this bug](https://github.com/dylang/node-rss/issues/40)

**Comment by fwenzel** [comment]
Oh hey thanks for pointing it out.

**Comment by rtorr** [comment]
Could something like `/*eslint-disable*/` and `/*eslint-enable*/` in the future be used?

**Comment by coleGillespie** [comment]
@rtorr yes we could fix this in es lint if we do not want to force camelCase. the exact fix in eslintrc would be 

```
camelcase: false
```

else fixing this will not pass lint

**Comment by jasonrhodes** [comment]
I think @rtorr is talking about temporarily turning off the lint rule for a given block of code, which you can do with jshint, etc. Is `/*eslint-disable*/` the syntax for that? I'm not sure.

**Comment by imakewebthings** [comment]
You can target a specific rule for enabling and disabling via comments. May I recommend only disabling the rules necessary until the upstream issue is resolved.

```
/* eslint camelcase: 0 */
...the code
/* eslint camelcase: 2 */

**Comment by rtorr** [comment]
@jasonrhodes yeah, that is what I mean't. It would be nice to use what we need over linting, for a block of code.
